Content.Shared/Doors/Systems/SharedAirlockSystem.cs

```diff
@@ -43,7 +43,7 @@ private void OnBeforeDoorClosed(EntityUid uid, AirlockComponent airlock, BeforeD
 
         if (TryComp(uid, out DoorComponent? door)
             && !door.Partial
-            && !CanChangeState(uid, airlock))
+            && DoorSystem.IsBolted(uid))
```

Comment:
If I read the code comment above correctly then `!door.Partial` should be there to bypass the power check in `CanChangeState`. So instead of removing this check and potentially breaking something else I would rather find the underlying cause for why that is not working.